.travis.yml

```diff
@@ -29,12 +31,18 @@ before_script:
     - cmake ../thirdparty/external
     - make -j4
     - cd ..
+    - if [ "$TRAVIS_OS_NAME" == "osx" ]; then export PATH=/usr/local/opt/qt5/bin:$PATH; fi
 
 script:
     # Build displaz
     - mkdir build
     - cd build
-    - cmake ..
+    - if [ "$TRAVIS_OS_NAME" == "linux" ]; then cmake -DDISPLAZ_USE_QT4=1 ..; fi
+    - if [ "$TRAVIS_OS_NAME" == "osx" ]; then cmake ..; fi
+    - make -j4
+    - make clean
+    - if [ "$TRAVIS_OS_NAME" == "linux" ]; then cmake -DCMAKE_BUILD_TYPE=Debug -DDISPLAZ_USE_QT4=1 ..; fi
+    - if [ "$TRAVIS_OS_NAME" == "osx" ]; then cmake -DCMAKE_BUILD_TYPE=Debug ..; fi
```

Comment:
This seems like something that the travis build matrix should take care of?  Maybe we can we get one dimension of the build matrix to set an environment variable for CMAKE_BUILD_TYPE.